api/serve.py

```
@@ -128,6 +131,64 @@ def _generate_smart_sql(question: str, schema: str) -> tuple[str, float, float]:
+                        return f"{col} = '{_format_val(val)}'"
+
+        for col in text_cols:
+            pattern = rf'\b{re.escape(col.lower())}\s+(?:is\s+|=\s*)?["\']?([A-Za-z]\w+)["\']?'
+            m = re.search(pattern, q)
+            if m:
+                val = m.group(1).strip()
+                if val.lower() not in skip_words:
+                    return f"{col} = '{_format_val(val)}'"
+
+        for col in text_cols:
+            pattern = rf'\b(?:named|called)\s+["\']?([A-Za-z]\w+)["\']?'
```

Ruff (0.15.6)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)